### ISSUE TITLE
wordpress-nginx Need add Start Of the Services on Reboot

### ISSUE THREAD

**Issue by pete-experimenter** [ansible/ansible-examples#93]
Provisioned the machine OK, which is awesome, but after the machine reboot everything is down. Need to add autostart of:
* nginx
* php-fpm

Something like this should do
```
- name: Autostart stuff
  shell: sudo /sbin/chkconfig --add {{ item }} && sudo /sbin/chkconfig {{ item }} on
with_items:
 - nginx
 - "php-fpm"
```

Again thanks for sharing the ansible scripts. Work like charm!

**Comment by stevemcquaid** [comment]
https://github.com/ansible/ansible-examples/pull/116

**Comment by varnav** [comment]
I've offered a simpler PR: https://github.com/ansible/ansible-examples/pull/122

**Comment by varnav** [comment]
Can we consider this fixed?

**Comment by gregdek** [comment]
Yes, since we merged #122.

**Comment by gregdek** [comment]
Thanks!
